lib/queue.js

```diff
@@ -45,8 +45,13 @@ class Queue extends Emitter {
       });
     }
 
-    this._delayedTimer = new EagerTimer(this.settings.nearTermWindow);
-    this._delayedTimer.on('trigger', this._activateDelayed.bind(this));
+    // To avoid changing the hidden class of the Queue.
+    this._delayedTimer = this.settings.activateDelayedJobs
```

Comment:
I think this could just be
```js
if (this.settings.activateDelayedJobs) {
  this._delayedTimer = new EagerTimer(this.settings.nearTermWindow);
  this._delayedTimer.on('trigger', this._activateDelayed.bind(this));
}
```
The `if (this._delayedTimer) {` check will still work the same way with _delayedTimer just being undefined instead of null.